tests/Entity/UserCreationTraitTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace weitzman\DrupalTestTraits\Tests\Entity;
+
+use Drupal\KernelTests\AssertLegacyTrait;
+use Drupal\Tests\RandomGeneratorTrait;
+use Drupal\user\Entity\Role;
+use PHPUnit\Framework\TestCase;
+use weitzman\DrupalTestTraits\DrupalSetup;
+use weitzman\DrupalTestTraits\Entity\UserCreationTrait;
+
+/**
+ * Tests the user creation trait.
+ */
+class UserCreationTraitTest extends TestCase
+{
+    use DrupalSetup;
+    use UserCreationTrait;
+    use RandomGeneratorTrait;
+    use AssertLegacyTrait;
```

Comment:
are we using RandomGeneratorTrait and AssertLegacyTrait?

Reply:
Each of the core create methods utilizes `RandomGeneratorTrait`, and the core `UserCreationTrait` still calls into `AssertLegacyTrait::assertIdentical` when creating roles (which happens on `userCreate`).